base/v0_3/validate_test.go

```diff
@@ -228,3 +228,26 @@ func TestValidateDirMode(t *testing.T) {
 		})
 	}
 }
+
+// TestUnkownIgnitionVersion tests that butane will raise a warning but will not fail when an ignition config with an unkown version is specified
+func TestUnkownIgnitionVersion(t *testing.T) {
+	test := struct {
+		in      Resource
+		out     error
+		errPath path.ContextPath
+	}{
+		Resource{
+			Inline: util.StrToPtr(`{"ignition": {"version": "4.0.0"}}`),
```

Comment:
Hmm, just so we never have to touch this test, lets make the number negative or larger?